content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -18,19 +17,46 @@ Kubernetes DNS schedules a DNS Pod and Service on the cluster, and configures
 the kubelets to tell individual containers to use the DNS Service's IP to
 resolve DNS names.
 
-### What things get DNS names?
-
 Every Service defined in the cluster (including the DNS server itself) is
-assigned a DNS name.  By default, a client Pod's DNS search list will
-include the Pod's own namespace and the cluster's default domain.  This is best
-illustrated by example:
+assigned a DNS name. By default, a client Pod's DNS search list will
+include the Pod's own namespace and the cluster's default domain. 
+
+### Namespaces of Services 
+
+DNS queries for services change based on namespace of the pod. DNS queries
+that don't specify a namespace are limited to the pod's namespace. Access
+services in other namespaces by including the namespace in the DNS query. 
```

Comment:
I am trying to walk through the example/description.

DNS queries for services change [from what? I'm not sure what the change is] based on the namespace of the pod.

nit: You can access services in other namespaces by including the namespace in the DNS query?

Reply:
how about:

> A DNS query may return different results based on the namespace of the pod making 
> it. DNS queries that don't specify a namespace are limited to the pod's namespace.